diesel/src/query_builder/bind_collector.rs

```diff
@@ -7,20 +7,22 @@ pub trait BindCollector<DB: Backend> {
 
 #[derive(Debug)]
 pub struct RawBytesBindCollector<DB: Backend + TypeMetadata> {
-    pub binds: Vec<(DB::TypeMetadata, Option<Vec<u8>>)>,
+    pub metadata: Vec<DB::TypeMetadata>,
+    pub binds: Vec<Option<Vec<u8>>>,
 }
 
 impl<DB: Backend + TypeMetadata> RawBytesBindCollector<DB> {
     pub fn new() -> Self {
```

Comment:
Did clippy not suggest to derive Default here? Maybe because of the type bounds?